src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java

```diff
@@ -49,6 +68,14 @@ public String getName() {
         return this.name;
     }
 
+    public void setKeyStorePath(String ksp) {

```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: com.puppycrawl.tools.checkstyle.checks.FinalParametersCheck
**Severity**: ERROR
**File**: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerColumnEncryptionCertificateStoreProvider.java L71

Parameter ksp should be final.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1413975493>*